benchmark/CMakeLists.txt

```diff
@@ -12,4 +12,5 @@ target_include_directories(ubench PRIVATE ${UMF_CMAKE_SOURCE_DIR}/include/)
 target_link_libraries(ubench
 	unified_memory_framework
 	numa
```

Comment:
@vinser it is interesting - can you elaborate more on your statement?

Reply:
I meant the following:

- if we build UMF as a static library then all libumf dependencies should be linked with the target binary that links with UMF, right?
- if we build UMF as a dynamic library then libumf is linked with libnuma.

My question is does ubench depend on libnuma or we are linking libnuma because libumf has a dependency on it?

Reply:
Yes, it should.

Reply:
We are linking with libnuma because libumf has a dependency on it.
Fixed.
Done.